about.css

```diff
@@ -0,0 +1,472 @@
+.past2 h3 {
+  font-size: 25px;
+  margin: 0.5rem 0;
+}
+
+.past1 p,
+.past2 p {
+  font-size: 18px;
+  color: white;
+  text-align: center;
+}
+
+.desktop {
+  display: none;
+}
+
+.sponsors {
+  background-image: url(images/footer.jfif);
+}
+
+.sponsors h2 {
+  text-align: center;
+  color: #d3d3d3;
+}
+
+.sponsors .contain {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.imgg {
+  width: 100px;
+  margin: 10px;
+}
+
+.sponsors .contain img {
+  width: 100%;
+}
+
+.con {
+  display: flex;
+  padding: 1rem 2rem;
+  align-items: center;
+  background-size: cover;
+  color: white;
+}
+
+@media (min-width: 768px) {
+  .container {
+    width: 75%;
+    margin: 0 auto;
+  }
+
+  header {
+    background-image: url(images/bgtest.jpg);
+    padding: 0;
+  }
+
+  .icon {
+    display: none;
+  }
+
+  .upper {
+    background-color: rgb(66, 66, 66);
+    display: block;
+  }
+
+  .socials {
+    width: 75%;
+    margin: 0 auto;
+    color: white;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    padding: 1rem 0;
+  }
+
+  nav li {
+    margin-left: 10px;
+    margin-bottom: 0;
+  }
+
+  .socials li {
+    margin-left: 1rem;
+    font-size: 16px;
+  }
+
+  .socials i {
+    color: white;
+    font-size: 16px;
+  }
+
+  .logo {
+    display: block;
+  }
+
+  .links {
+    background-color: white;
+    padding: 0;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 75%;
+    margin: 0 auto;
+  }
+
+  nav ul {
+    position: unset;
+    display: flex;
+    align-items: center;
+    height: auto;
+    width: auto;
+    background-color: transparent;
+    padding: 0.5rem 0;
+  }
+
+  nav a {
+    width: 100px;
+  }
+
+  nav li a {
+    font-size: 18px;
+  }
+
+  nav a img {
+    width: 100%;
+  }
+
+  #larger {
+    border: 2px solid #ec5242;
+    padding: 0.5rem;
+    color: #ec5242;
+    display: block;
+  }
+
+  #larger a {
+    color: #ec5242;
+  }
+
+  .intro {
+    width: 75%;
+    margin: 0 auto;
+    padding: 5rem 0;
+  }
+
+  .intro h1 {
+    font-size: 45px;
+    text-transform: uppercase;
+  }
+
+  .big {
+    width: 700px;
+    margin: 0 auto;
+  }
+
+  section,
+  .ntslogo {
+    width: 70%;
+    margin: 0 auto;
+  }
+
+  .ntsimage {
+    padding: 0 0.5rem;
+  }
+
+  footer p {
+    margin-left: 0;
+  }
+
+  .ntslogo p {
+    margin: 1.5rem 0;
+  }
+
+  .wrap {
+    display: flex;
+    justify-content: space-between;
+    margin: 2rem 0;
+  }
+
+  .pastsummits {
+    width: 75%;
+    margin: 0 auto;
+  }
+
+  .past1,
+  .past2 {
+    width: 48%;
+    padding: 1rem;
+  }
+
+  .desktop {
+    display: block;
+  }
+
+  .mobile {
+    display: none;
+  }
+
+  .sponsors {
+    display: block;
+    width: 100%;
+  }
+
+  .sponsors h2 {
+    margin: 2rem 0;
+  }
+
+  footer {
+    padding: 2rem 0;
+  }
+
+  .con {
+    width: 75%;
+    margin: auto;
+  }
+
+  .desktop,
+  footer {
+    display: block;
+  }
```

Comment:
- [ ] The `background-color` for the footer of the about page is black and not white on desktop view. It is only on the mobile view that it should be white. Kindly implement this change. 👍 

### Template Design

![image](https://user-images.githubusercontent.com/77106962/210669686-a3d34d3b-fda0-4fab-8610-6426e2eebd12.png)